week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -11,15 +11,31 @@
 
  */
 
+
+
 class ATM {
     // Add your code here
-
+constructor(){
+    this.amount = 100.0
+}
+make_deposit(num){
+    this.amount += num
+}
+make_withdrawl(num){
+    
+ if ((this.amount - num) > 0) {this.amount -= num}
+ else {console.log("Sorry you don't have enough balance to withdrawl this amount!");
+}
+    }
+    check_balance(){
+        console.log(this.amount);   
+    }
```

Comment:
Great work well done! You could maybe tidy up some of your indentation but apart from that this is great :)